src/map/map.ts

```diff
@@ -1086,6 +1132,27 @@ export class PropertiesMap {
         this._plot.addEventListener('wheel', () => {});
     }
 
+    /** Validate provided min max values */
+    private _getAxisRange = (
+        min: number,
+        max: number,
+        axisName: string
+    ): [number | undefined, number | undefined] => {
+        const minProvided = !isNaN(min);
+        const maxProvided = !isNaN(max);
+
+        // At least one range value is specified. By default, zeros are set
+        if (minProvided && maxProvided) {
+            if (min <= max) {
+                return [min, max];
+            }
+            sendWarning(
+                `The inserted min and max values in ${axisName} are such that min > max! The default values will be used.`
+            );
```

Comment:
Is this actually an issue? Or does plotly switch the order and goes high to low?

Reply:
Yes, if min remains bigger than max and user tries to change that range in the popup, it raises `Maximum call stack overflow` error due to this function 

https://github.com/lab-cosmo/chemiscope/blob/47770ab70bee39995da479bcd68df7aca7b3103c/src/map/map.ts#L723-L736

(I suppose it does reset() to the previous value which is still min > max => loop)

Reply:
👍